src/rules/handlers.ts

```diff
@@ -44,9 +46,9 @@ export function buildHandler
 }
 
 const handlerBuilders: { [T in HandlerType]: HandlerBuilder<HandlerDataLookup[T]> } = {
-    simple: ({ data, status }: SimpleHandlerData): RequestHandler => {
+    simple: ({ data, status, headers }: SimpleHandlerData): RequestHandler => {
         let responder = _.assign(async function(request: OngoingRequest, response: express.Response) {
-            response.writeHead(status);
+            response.writeHead(status, headers);
             response.end(data || "");
         }, { explain: () => `respond with status ${status}` + (data ? ` and body "${data}"` : "") });
```

Comment:
The headers aren't included in the explanation for this rule. Can you expand this, so if headers are set it includes them here in some sensible format, as it does with the body?